### DISCUSSION TITLE
Phase locking between two lasers

### DISCUSSION THREAD

**Discussion by ZhimingShi** [linien-org/linien#385] [Q&A]
Hello！I want to use Linien to lock the phase of two lasers, can this software do that? For example, lock the laser to a frequency comb. The laser is combed with a frequency comb to obtain an RF signal less than 30MHz. The phase of the RF signal is then demodulated using Linien to obtain an error signal. The error signal is processed by PID and fed back to the laser for phase locking.

**Comment by bleykauf** [comment]
Phase-frequency detection is currently not possible. You could, however, do the phase-frequency detection externally and use Linien as a lockbox. While I do not have any experience with it myself, [pyrpl](https://pyrpl.readthedocs.io/en/latest/) might be more suitable for this application.
